benchs/hash.ts

```diff
@@ -0,0 +1,80 @@
+import { webcrypto } from "node:crypto";
```

Comment:
you can create .bench.ts files alongside the file that exports your function that you're going to test. I think this is the standard adopted by deno community